### PR TITLE
Fixed function documents for GIRFApplier and GIRFEssential

### PR DESCRIPTION
**Pull request by nbwuzhe**
Fixed function documents for GIRFApplier and GIRFEssential

### CODE REVIEW

**Comment by alexjaffray** [comment]
apply_girf still needs documentation. I'll add tomorrow morning

File: src/Util.jl [hunk 1]
```diff
@@ -267,7 +267,18 @@ end
...
 # end
 
-## Get gradients from the trajectory
+"""
+    nodes_to_gradients(nodes::Matrix; gamma=42577478, dwellTime=2e-6, FOV=[220,220,1],reconSize=[200,200,1])
+
+Function to calculate the gradient train used to acquire k-space data
+
+# Arguments
+* `nodes::Matrix` - input k-space nodes traversed in order, acquired with constant dwell time.
```
Comment (nbwuzhe) [inline src/Util.jl]: Hi Alex, could you please specify the size of this input argument?

File: src/Util.jl [hunk 2]
```diff
@@ -267,7 +267,18 @@ end
...
+
+Function to calculate the gradient train used to acquire k-space data
+
+# Arguments
+* `nodes::Matrix` - input k-space nodes traversed in order, acquired with constant dwell time. 
+* `gamma` - gyromagnetic ratio
+* `dwellTime` - the dwell time of the trajectory
+* `FOV` - spatial field of view in [mm]
+* `reconSize` - reconstruction matrix size
+"""
```
Comment (nbwuzhe) [inline src/Util.jl:287]: Also the outputs and their sizes?

**Review by alexjaffray — APPROVED** [review]
Great additions, documentation is more complete now.

**Pull request merged**